tic_tac_toe/game.py

```diff
@@ -63,22 +95,60 @@ def _check_winning_combinations(board, player):
     Returns the player (winner) of any of the winning combinations is completed
     by given player, or None otherwise.
     """
-    pass
+    # positions = [
```

Comment:
When you are "presenting" your code, it's always bad to push commented or incomplete code :)